helm/values-notprod.yaml

```diff
@@ -0,0 +1,14 @@
+---
+hocs-backend-service:
+  minReplicas: 1
+  maxReplicas: 1
+
+  deploymentAnnotations:
+    downscaler/downtime: "Mon-Sun 00:00-07:55 Europe/London,Mon-Sun 18:05-24:00 Europe/London"
+
+  app:
+    resources:
+      limits:
+        memory: 512Mi
```

Comment:
Request and limit being the same for RAM is probably not intended.
Request should be set to slightly above the "normal" usage, and limit set to the lowest unreasonable amount of RAM usage possible.

Here (...I think...) if a container demands more than 512Mi it will get OOM-killed, which would not be ideal considering you're also specifying there should only be one pod at any given time.

Reply:
This is deliberate, and it's the default behaviour when limits aren't set, but our namespaces have default limits too so we have to do this. We can look at this outside of this PR, but we were getting too many restarts when nodes could provide the request but not the limit, so for now we can set a guaranteed size and let the JMV exist within that.